src/main/java/lembas/core/api/common/SimpleLogger.java

```diff
@@ -0,0 +1,71 @@
+/*******************************************************************************
+ * Copyright (C) 2020 Lembas Modding Team
+ * 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ ******************************************************************************/
+package lembas.core.api.common;
+
+import java.util.Objects;
+import java.util.function.Function;
+
+/**
+ * A simple wrapper for a logger using String.format for formatting the log
+ * messages. The [...]Throwable functions are there to avoid bugs related to the
+ * varargs.
+ *
+ */
+public abstract class SimpleLogger

```

Comment:
This could be transformed into an interface for a little more flexibility.

Reply:
In the PvP Mode Mod it actually is an interface, but then I have to create separate factory classes, because interfaces don't support private members. Therefore I made the class abstract (which btw. shouldn't be a huge downside because the only implementation is probably still the one in our internal package...)